app/controllers/questionnaires_controller.rb

```diff
@@ -16,8 +16,12 @@ def update
 
 protected
 
+  def module_item
+    @module_item ||= ModuleItem.find_by(training_module: module_params['training_module_id'], name: module_params['id'])
```

Comment:
This would be better as `questionnaire.module_item` IMO

Reply:
I was trying to be consistent - principle of least surprise